docs/SOPs/contributor_matrix_addition_SOP.md

```diff
@@ -27,8 +27,25 @@ Where either the expression matrix or cell type annotations cannot be found, the
 
 ## Filling in metadata about the files
 
+For each expression matrix or cell type annotation file that is found, a row needs to be filled in the metadata spreadsheet, in the ‘Analysis file’ tab. Analysis files can be linked to sequence file or biomaterial entities via the ‘Analysis_protocol’ entity (see the ‘Analysis protocol’ tab). This functions similar to a ‘sequencing protocol’ to link different entities. 
```

Comment:
The wording is confusing here; I think you were trying to go for "Analysis files can be linked to sequence files or biomaterial entities via processes; This is done in the spreadsheet in the same way that other entities are linked. Information related to the analysis protocol is captured in the `Analysis_protocol` entity (See the `Analysis protocol` tab) linked to the process."

Reply:
Thank you, that was much more eloquently put!